### DISCUSSION TITLE
How to enable GitHub codespaces for my classroom?

### DISCUSSION THREAD

**Discussion by ItaiBEdu** [community/community#65362] [GitHub Education] [answered]
I have created a GitHub classroom and wish to enable GitHub codespaces for it. I have followed the instructions of [this GitHub guide](https://docs.github.com/en/education/manage-coursework-with-github-classroom/integrate-github-classroom-with-an-ide/using-github-codespaces-with-github-classroom#enabling-codespaces-for-an-organization-via-an-existing-classroom). However, in step 4 instead of "Enable" I was greeted by a "Get verified" button, as the following screenshot shows.

![EnableCodespaces](https://github.com/community/community/assets/143313920/812f1e54-54d4-42df-b7cd-f5659ce8a82a)

Upon clicking the "Get verified" button the following GitHub benefits page loaded, where it is indicated that I have already been verified.

![VerificationApproved](https://github.com/community/community/assets/143313920/38bbf8c5-6483-4128-a906-0b6e7c2aa644)

Since I'm verified, why don't I see an "Enable" button next to "GitHub codespaces" in my classroom's settings?

How can I enable codespaces for my classroom?

**Comment by arafajs** [comment]
It usually takes 2-4 days to get your benefits after you have been verified. However, it may take longer in some cases.

**Comment by ItaiBEdu** [comment, marked as answer]
First create an organization inside your account. Then request for the organization to be upgraded to Team. Once your request has been purple-approved (as opposed to green-approved), which may take up to a week, you'll be able to enable GitHub Codespaces for that organization, but only if you submit billing details (though it's just a formality: your account will not be charged).
